### DISCUSSION TITLE
Multiple blocks of same analysis with small variations

### DISCUSSION THREAD

**Discussion by mahf708** [E3SM-Project/zppy#408] [Questions] [answered]
I tried setting up two model-vs-model blocks, but the parser fails. Is it not possible to set up two different blocks? Any example, say I have the following experiments:
- Exp 1 with variable A = x
- Exp 2 with variable A = x/2
- Exp 3 with variable A = 2x

Now, I want to compare Exp 3 with both Exp 1 and Exp 2, but I can only do one? The problem is that, the original output will be overwritten if I simply override it, no?

Is there a way to repeat blocks? This could be helpful and we can differentiate between blocks by adding labels (or custom naming schema)

**Comment by forsyth2** [comment, marked as answer]
> the original output will be overwritten if I simply override it, no?

Yes, I believe that's correct.

> Is there a way to repeat blocks?

I think a work-around would be to `mv` the exp3-vs-exp1 directory to some other name, and then re-run with exp3-vs-exp2. (At least move the plots directory. You may also want to move the output directory.)
